dashboard/src/components/BootsTable/BootsTable.tsx

```diff
@@ -150,7 +151,46 @@ const TableCellComponent = ({
   );
 };
 
+const TableRowComponent = ({
+  row,
+  index,
+  currentLog,
+  getRowLink,
+  openLogSheet,
+}: {
+  row: Row<TestByCommitHash>;
+  index: number;
+  currentLog?: number;
+  getRowLink: (testId: TestHistory['id']) => LinkProps;
+  openLogSheet: (index: number) => void;
+}): JSX.Element => {
+  const className = useMemo(() => {
+    let initialClassName = 'cursor-pointer hover:bg-lightBlue';
+
+    if (typeof currentLog === 'number' && index === currentLog)
```

Comment:
we don't need to do `typeof currentLog === 'number'` all the time, if index is a number `index === undefined` will be false